proto/greenfield/sp/events.proto

```diff
@@ -28,6 +28,8 @@ message EventCreateStorageProvider {
   Status status = 8;
   // description defines the description terms for the storage provider
   Description description = 9 [(gogoproto.nullable) = false];
+  // seal_bls_key defines the bls pub key owned by storage provider used when sealing object
```

Comment:
Need to add id to the event.

Reply:
@alexgao001 Could you help to add it to the event?